tests/integration/test_dynamodb.py

```diff
@@ -1342,33 +1348,7 @@ def test_dynamodb_stream_to_lambda(
         lambda_client.delete_event_source_mapping(UUID=mapping_uuid)
 
     @pytest.mark.aws_validated
-    def test_dynamodb_batch_write_item(
```

Comment:
was the test case `test_dynamodb_batch_write_item removed`?

Reply:
Yes I removed it. It has does almost exactly the same as `test_batch_write_items()`. Both target the function `dynamodb_client.batch_write_item()`...
`test_dynamodb_batch_write_item()` tests 3 put requests
`test_batch_write_items()` tests 1 delete- and 1 putrequest
Let me know if I'm missing something :)

Reply:
ah I see - seems to be very similar indeed :) 
/cc @giograno

Reply:
Fine with dropping one :)